packages/@react-spectrum/color/stories/ColorArea.stories.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {ColorArea, ColorSlider} from '../';
+import {ColorChannel, SpectrumColorAreaProps} from '@react-types/color';
+import {Flex} from '@adobe/react-spectrum';
+import {Meta, Story} from '@storybook/react';
+import {parseColor} from '@react-stately/color';
+import React, {useState} from 'react';
+import {Text} from '@react-spectrum/text';
+
+
+const meta: Meta<SpectrumColorAreaProps> = {
+  title: 'ColorArea',
+  component: ColorArea
+};
+
+export default meta;
+
+const Template: Story<SpectrumColorAreaProps> = (args) => (
+  <ColorAreaExample {...args} />
+);
+
+let RGB: Set<ColorChannel> = new Set(['red', 'green', 'blue']);
+let difference = (a, b): Set<ColorChannel> => new Set([...a].filter(x => !b.has(x)));
+
+function ColorAreaExample(props: SpectrumColorAreaProps) {
+  let {xChannel, yChannel, isDisabled} = props;
+  let channels = new Set([xChannel, yChannel]);
+  let zChannel: ColorChannel = difference(RGB, channels).keys().next().value;
+  let [color, setColor] = useState(props.defaultValue || parseColor('#ff00ff'));
+  return (<div role="group" aria-label="RGB Color Picker">
+    <Flex gap="size-500" alignItems="center">
+      <Flex direction="column" gap="size-50" alignItems="center">
+        <ColorArea
+          {...props}
+          value={color}
+          onChange={(e) => {
+            if (props.onChange) {
+              props.onChange(e);
+            }
+            setColor(e);
+          }} />
+        <ColorSlider
```

Comment:
Perhaps out of scope, but noticed that VoiceOver doesn't announce the color as you adjust the ColorSlider. ColorArea announces "blue X, green Y" as you move the ColorArea slider around, should ColorSlider announce "red  Z" when its slider is moved?